src/reducers/books.js

```diff
@@ -0,0 +1,19 @@
+const booksReducer = (state, action) => {
+  if (action.type === 'CREATE_BOOK') {
+    const allBooks = [...state.books, action.book];
+    return {
+      ...state,
+      books: allBooks,
+    };
+  }
```

Comment:
- [ ] The initial state was created but it's not being used, we need to use it. It's best to define the default state here and assign to the state parameter as a default value. 
```JS
const booksReducer = (state = initState, action) => {
  if (action.type === 'CREATE_BOOK') {
    const allBooks = [...state.books, action.book];
    return {
      ...state,
      books: allBooks,
    };
  }
```

Reply:
Declaring the `initState` in the **_src/index.js_** file and using it in the _**reducers/books.js**_ file throws an error in the App, I don't know why the Project requirements want us to do it that way, that's why I left it out earlier, but its back in now.